config.md

```diff
@@ -308,15 +308,23 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value are strings.
+Keys MUST be unique, and best practice is to namespace the keys.
+Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
+If there are no annotations then this property MAY either be absent or an empty map.
+Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
```

Comment:
“MUST NOT generate an error” doesn't say how they *should* handle unknown entries.  I think we should [follow HTML's][1]:

> The [`HTMLUnknownElement`][2] interface must be used for HTML elements that are not defined by this specification (or other applicable specifications).

In our case, maybe wording like:

> Implementations consuming this configuration MAY log unrecognized annotation keys but MUST otherwise ignore them.

[1]: https://www.w3.org/TR/html5/dom.html#elements-in-the-dom
[2]: https://www.w3.org/TR/html5/dom.html#htmlunknownelement